src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/EJSONWrapper.java

```diff
@@ -0,0 +1,54 @@
+package io.stargate.sgv2.jsonapi.api.model.command.clause.filter;
```

Comment:
Added in this package only because `JsonLiteral` and `JsonType` are already here; seems better to co-locate than try to find otherwise better package match.